### ISSUE TITLE
Another line on Shout description window for moving translations to

### ISSUE THREAD

**Issue by Shizof** [Odie/skyui-vr#16]
Not an issue, just a request.

In this mod: https://www.nexusmods.com/skyrimspecialedition/mods/18572
I added pronunciations of shouts and put the translations in parenthesis.
It would be better if there was another line for the translations so they won't be so cluttered together.

Something like this: https://i.imgur.com/FW4TFt0.jpg

If it automatically splits the text to the other line if it has "|" (pipe) for example, it would be nice.

**Comment by Odie** [comment]
Wow, looks like you already a working mod! So, are you mostly looking to have slightly better looking layout?

I’m away from the computer for a bit so I can’t try things out directly ATM. I think the first thing I’d try is to insert a newline character between the pronouciation and the translated text. Assuming the texfield text alignment is already “centered” and multiline is enabled, this will probably just work. Hopefully, all the text would be lined up automatically, and we won’t have to add more code to tweak the alignment at runtime.

Some slight adjustment will likely be needed so the 2nd line isn’t overlapping other UI elements.

**Comment by Shizof** [comment]
Yeah I made that mod 2 months ago, but it felt incomplete and I didn't know it's possible to edit the ui that way until you came and started doing ui work. 
Inserting a new line character would probably be enough. If there are two spaces, the split would need to be on the last one.

**Comment by Odie** [comment]
Okay... I'll try to get to this tomorrow.

**Comment by Odie** [comment]
Btw, what do you think about having the dragon text fade out and replaced by the pronunciation after a short delay?

**Comment by Shizof** [comment]
Cool idea indeed.

**Comment by Odie** [comment]
Want to give this a go and see how you like it? This one has a 1.5 second delay, before the dragonscript is faded out and replaced by its pronunciation. The fade in/out is done over 0.5s.

[SkyUI - VR - dragon pronunciation.zip](https://github.com/Odie/skyui-vr/files/2475088/SkyUI.-.VR.-.dragon.pronunciation.zip)

There is just a single updated itemcard.swf in there.

**Comment by Shizof** [comment]
It's very good! I'll make a new version of my mod so this works better.( Not really necessary anymore but I'll change stuff like R1n to Raan and N7 to Nir) You used "Full Name" I'm guessing right?

**Comment by Odie** [comment]
from Shizof on discord:
```
String replace is the way to go(edited)
Numbers are going to be replaced with the corresponding strings:(edited)
1: aa
2: ei
3: ii
4: ah
6: ur
7: ir
8: oo
9: ey
this is it
it works for dlcs and thunderchild shouts as well
only thing is if these numbers come as the first letters you need to capitalize the first letters.
for example 3z = Iiz and 4 = Ah
Other than that it's normal replace. D4 = Dah
```

**Comment by Odie** [comment]
Here's a version that implements the discussed string replacement. Seems to work okay. =)

[SkyUI - VR - dragon pronunciation - 2.zip](https://github.com/Odie/skyui-vr/files/2476389/SkyUI.-.VR.-.dragon.pronunciation.-.2.zip)

**Comment by Shizof** [comment]
Yes. It works very well. Awesome job!

**Comment by Odie** [comment]
Great. This will go out with the next version then.
I hope people won't be too disturbed by this unexpected addition.

Implemented by d4fed7c81a8e7794b53b1621378b580745704250
